cfgov/unprocessed/css/video-player.less

```diff
@@ -55,6 +55,8 @@
     // @black not used because it's not totally black.
     background-color: #000;
     text-align: center;
+    height: 100%;
+    width: 100%;
```

Comment:
Shoot. This doesn't but adding the featured image class does. Now we need to figure out how to segment those.